internal/util/reports_test.go

```diff
@@ -245,11 +245,16 @@ func getTLSListener() (net.Listener, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Most TLS configurations in this project use TLS 1.2 for FIPS mode compatibility
+	// This does not because it causes an unexplained test failure where the test clients do report sending data
+	// but the test server receives only an EOF/connection closed without any data. This only occurs in GitHub Actions
+	// As this test server is not shipped as part of the product, allowing 1.3 here does not affect FIPS compatibility
```

Comment:
Seems like we should create some kind of issue for this, perhaps even an upstream issue for Actions?

Reply:
If we wanted to pursue it yes, but I don't think we want to given how little we can definitively say happens in Actions to cause it and the minimal benefit of fixing it.